cmd/kueue/main.go

```diff
@@ -276,17 +276,21 @@ func setupControllers(ctx context.Context, mgr ctrl.Manager, cCache *cache.Cache
 	}
 
 	// setup provision admission check controller
-	if features.Enabled(features.ProvisioningACC) && provisioning.ServerSupportsProvisioningRequest(mgr) {
-		// A info message is added in setupIndexes if autoscaling is not supported by the cluster
-		ctrl, err := provisioning.NewController(mgr.GetClient(), mgr.GetEventRecorderFor("kueue-provisioning-request-controller"))
-		if err != nil {
-			setupLog.Error(err, "Could not create the provisioning controller")
-			os.Exit(1)
-		}
+	if features.Enabled(features.ProvisioningACC) {
+		if err := provisioning.ServerSupportsProvisioningRequest(mgr); err != nil {
+			setupLog.Info("Skipping provisioning controller setup: Provisioning Requests not supported (Possible cause: missing or unsupported cluster-autoscaler)")
+		} else {
+			// A info message is added in setupIndexes if autoscaling is not supported by the cluster
```

Comment:
nit: I think we can safely drop this comment. The same info is added in both places so the comment is a bit weird